adl_parser.py

```diff
@@ -0,0 +1,114 @@
+block_names = ("object", "region", "info", "table ")
```

Comment:
should only be in one place..please remove from here and make sure you keep the most recent version of this code